go.mod

```diff
@@ -1,50 +1,32 @@
 module github.com/Scalingo/go-plugins-helpers
 
-go 1.20
+go 1.23.5
 
 require (
-	github.com/Microsoft/go-winio v0.6.1
-	github.com/Scalingo/go-handlers v1.8.1
-	github.com/Scalingo/go-utils/logger v1.2.0
-	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
+	github.com/Microsoft/go-winio v0.6.2
+	github.com/Scalingo/go-handlers v1.8.2
+	github.com/Scalingo/go-utils/logger v1.4.0
+	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/docker/docker v23.0.8+incompatible
```

Comment:
I'll update this dependency in a dedicated PR as it requires multiple new major versions.